components/nexus-httpclient/src/main/java/org/sonatype/nexus/httpclient/config/SecondsDeserializer.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.httpclient.config;
+
+import java.io.IOException;
+
+import org.sonatype.sisu.goodies.common.Time;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+/**
+ * {@link Time} in seconds deserializer.
+ *
+ * @since 3.0
+ */
+public class SecondsDeserializer
+    extends StdDeserializer<Time>
+{
+
+  public SecondsDeserializer() {
+    super(Time.class);
+  }
+
+  @Override
+  public Time deserialize(final JsonParser parser, final DeserializationContext context)
+      throws IOException
+  {
+    return Time.seconds(parser.readValueAs(Integer.class));
+  }
```

Comment:
BTW one of the advantages to Time is that the configuration value can specific the units.

So if I wanted to say 60 seconds I could say "60 sec" or I could also say "1 min" both when asked for the "seconds" value will be identical.

We may want to consider, if we are going to use more Time for UI configuration, to implement the ability to allow users to specify the raw Time value with some validation, instead of keeping the UI units static.